sql/2019/12_Mobile_Web/12_12.sql

```diff
@@ -0,0 +1,30 @@
+#standardSQL
+
+# input types
+
+# dataset: `httparchive.pages.2019_07_01_mobile`
+# sample: `httparchive.almanac.pages_desktop_1k`
+
+CREATE TEMPORARY FUNCTION parseStructuredData(payload STRING)
```

Comment:
Nit: this UDF isn't really parsing structured data. Maybe rename to `getInputTypes`?